subsys/mgmt/mcumgr/smp/src/smp.c

```diff
@@ -191,14 +191,27 @@ static int smp_handle_single_payload(struct smp_streamer *cbuf, const struct smp
 		bool ok;
 
 		*handler_found = true;
+#if defined(CONFIG_MCUMGR_MGMT_CUSTOM_PAYLOAD)
+		if (!handler->use_custom_payload) {
```

Comment:
Would just put the `if()` and `}` in `#if` here so there is not duplicated code

Reply:
Done